### PR TITLE
fix(styles): replace lead placeholders with mixins

### PR DESCRIPTION
**Pull request by oliverschuerch**

### CODE REVIEW

**Comment by swisspost-devs** [comment]
Preview environments are getting posted here, as soon as they are ready!

**Review by gfellerph — CHANGES_REQUESTED** [review]
Could you add a changeset?

File: .changeset/five-months-unite.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@swisspost/design-system-styles': patch
+---
+
+Replaced scss placeholder statements for .lead selector with pure mixins and variables.
```
Comment (gfellerph) [inline .changeset/five-months-unite.md:5]:
```suggestion
Fixed an issue with the font size of lead text that was too small due to a specificity issue with sass placeholders.
```

**Review by gfellerph — APPROVED** [review]
I'd change the changeset text a little to make it easier to understand for users.

**Pull request merged**